components/auth/apiKeysService.js

```diff
@@ -0,0 +1,15 @@
+const MongoLib = require('../../lib/mongo');
+
+class ApiKeysService {
+  constructor() {
+    this.collection = 'apiKeys';
```

Comment:
use config.dbCollections.apikeys, and register the collection in config file

Reply:
Done please check